scripts/install_script_ubuntu.sh

```diff
@@ -42,6 +49,14 @@ then
   GRR_TESTING=1;
 fi
 
+if [ "$1" == "--build-deps" ] || [ "$2" == "--build-deps" ]; then
```

Comment:
should we do this in a while / shift loop? install.sh --build-deps --localtest would already not work...